apiserver/facades/client/application/application.go

```diff
@@ -185,19 +168,19 @@ func newFacadeBase(stdCtx context.Context, ctx facade.ModelContext) (*APIBase, e
 		ctx.Clock(),
 	)
 
+	// TODO: replace nil caasBroker when its logic is migrated to the domain service.
```

Comment:
Can you put this TODO where the CAAS broker is actually being used to check for storage classes? That is to say if you follow it through to the point where it is used that is where this TODO should be.

Reply:
Looks like you have done that. I am happy for this TODO to be removed as we only need the one.